django_tables2/columns/base.py

```diff
@@ -239,6 +239,18 @@ def render(self, value):
         """
         return value
 
+    def order(self, queryset, is_descending):
+        """
+        Returns the queryset of the table.
+
+        This method can be overridden by :ref:`table.order_FOO` methods on the
```

Comment:
I would say to either add some documentation or do not reference it.

Of course, more documentation is better.